package.json

```diff
@@ -78,5 +78,5 @@
   "main": "index.js",
   "repository": "https://github.com/gunet/wallet-frontend",
   "author": "gregory1996 <grigoriskatrakazas@gmail.com>",
```

Comment:
```suggestion
  "repository": "https://github.com/wwwallet/wallet-frontend",
  "author": "",
```

Reply:
I'm unsure whether it's appropriate to include these changes in this pull request or if they should be committed in a separate one.

Reply:
It's off topic given the original scope, but I don't mind expanding the scope for something this small. :slightly_smiling_face: